tests/acceptance/features/apiGraph/assignRole.feature

```diff
@@ -0,0 +1,33 @@
+@api
+Feature: assign role
+  As an admin, I want to assign roles to users.
+  I cannot change my own role.
+  Users without an admin role cannot get the list of roles, assignments list and assign roles to users
```

Comment:
how about adding scenarios related to `a normal user tries to get roles, assign roles and get list of assignments` if missing.

Reply:
no not needed such scenario as confirmed by kulman